### DISCUSSION TITLE
Reflections on Progress and Motivation for the Journey Ahead

### DISCUSSION THREAD

**Discussion by mrkubernetes** [Cyfrin/foundry-full-course-cu#2546] [Q&A]
How has your experience been learning Solidity so far, and what keeps you motivated or gives you hope as you move forward?

**Comment by EngrPips** [comment]
The only motivation I have is I want to be exceptionally good and better than majority of people.

**Reply by mrkubernetes** [reply]
You don't have to care about the majority, life is a single player game. Strive to be better than you were yesterday. Keep grinding mate :)

**Reply by EngrPips** [reply]
🙇🏾‍♂️🙇🏾‍♂️🙇🏾‍♂️

**Comment by mumernisar** [comment]
THE INDOMITABLE HUMAN SPIRIT ☝️

**Comment by EngrPips** [comment]
Interesting.
